### PR TITLE
BZ1823977 - Better approval command for CSR

### PR DESCRIPTION
**Pull request by vwalek**

### CODE REVIEW

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -773,10 +773,10 @@ $ oc adm certificate approve <csr_name> <1>
 ----
 <1> `<csr_name>` is the name of a CSR from the list of current CSRs.
 
-* Approve all CSRs:
+* Approve only Pending CSRs:
```
Comment (bergerhoffer) [inline install_config/redeploying_certificates.adoc:776]: Suggest changing "only Pending" to "all pending"

**Review by bergerhoffer — COMMENTED** [review]
Just one minor suggestion

File: install_config/redeploying_certificates.adoc (hunk 1 above)
Comment (vwalek) [reply install_config/redeploying_certificates.adoc]: ok, done

**Comment by bergerhoffer** [comment]
@barleyer Can you verify that this checks out to approve only pending CSRs? Thanks!

**Pull request merged**